### ISSUE TITLE
Standalone Selenium Server not starting: Can't find bundle for base name javax.servlet.LocalStrings, locale de_AT

### ISSUE THREAD

**Issue by ThomasLandauer** [SeleniumHQ/selenium#3610]
## Meta -
OS:  
Windows 10
Selenium Version:  
Standalone Server 3.2.0

## Expected Behavior -
Server starting

## Actual Behavior -
Server not starting

## Steps to reproduce -
After installing Java 8 Update 121 I'm trying to start the Selenium server like this:
`java -jar "SeleniumStandaloneServer3.2.0.jar"`

But I'm getting this error:

````
18:57:42.820 INFO - Selenium build info: version: '3.2.0', revision: '8c03df6'
18:57:42.820 INFO - Launching a standalone Selenium Server
2017-03-06 18:57:42.836:INFO::main: Logging initialized @204ms to org.seleniumhq.jetty9.util.log.StdErrLog
18:57:42.883 INFO - Driver class not found: com.opera.core.systems.OperaDriver
18:57:42.883 INFO - Driver provider com.opera.core.systems.OperaDriver registration is skipped:
Unable to create new instances on this machine.
18:57:42.883 INFO - Driver class not found: com.opera.core.systems.OperaDriver
18:57:42.883 INFO - Driver provider com.opera.core.systems.OperaDriver is not registered
18:57:42.883 INFO - Driver provider org.openqa.selenium.safari.SafariDriver registration is skipped:
 registration capabilities Capabilities [{browserName=safari, version=, platform=MAC}] does not match the current platform WIN10
Exception in thread "main" java.lang.ExceptionInInitializerError
        at org.openqa.selenium.remote.server.SeleniumServer.boot(SeleniumServer.java:113)
        at org.openqa.grid.selenium.GridLauncherV3$1.launch(GridLauncherV3.java:238)
        at org.openqa.grid.selenium.GridLauncherV3.main(GridLauncherV3.java:99)
Caused by: java.util.MissingResourceException: Can't find bundle for base name javax.servlet.LocalStrings, locale de_AT
        at java.util.ResourceBundle.throwMissingResourceException(Unknown Source)
        at java.util.ResourceBundle.getBundleImpl(Unknown Source)
        at java.util.ResourceBundle.getBundle(Unknown Source)
        at javax.servlet.GenericServlet.<clinit>(GenericServlet.java:95)
        ... 3 more
````

**Comment by lmtierney** [comment]
You're requesting Safari on Windows10. You need to be on a mac to use Safari.

**Comment by lmtierney** [comment]
Sorry I misread the stack

**Comment by ThomasLandauer** [comment]
@lmtierney Sorry, don't know what you mean :-( What are capabilities?

**Comment by lmtierney** [comment]
That stacktrace is solely from executing `java -jar "SeleniumStandaloneServer3.2.0.jar"` ?

**Comment by ThomasLandauer** [comment]
Yes!

And the `.jar` file comes directly from http://www.seleniumhq.org/download/ (linktext "Download version 3.2.0")

**Comment by lmtierney** [comment]
I just tried this on Windows10 myself and it's working. What language is your Windows set to?

**Comment by ThomasLandauer** [comment]
German. This is probably where the `locale de_AT` in the error message comes from ;-)

**Comment by lmtierney** [comment]
Have you tried switching to EN to see if thats the issue?

**Comment by ThomasLandauer** [comment]
I just followed the instructions at https://www.java.com/en/download/help/locale.xml and switched to English (United States). I'm not sure what this option is supposed to do exactly - however, my Windows is still being displayed in German.

And I still get the same error message :-(

Since there are many places where you can change language-related stuff in Windows: Do you have a hint which one is the relevant setting?

**Comment by ThomasLandauer** [comment]
I somehow managed to switch to English :-) But that didn't change much - now the error message reads:  
`Caused by: java.util.MissingResourceException: Can't find bundle for base name javax.servlet.LocalStrings, locale en_US`

**Comment by ThomasLandauer** [comment]
I just tried with version 3.1.0 - same result :-(

**Comment by krmahadevan** [comment]
@ThomasLandauer - I am guessing that perhaps the jar file may have been corrupted. If you have firefox browser, then can you please open the firefox browser and open the below URL 

`jar:file:///<LocationOfSeleniumJarGoesHere>!/javax/servlet/` and see if you are able to see the `LocalStrings.properties` file in the jar ?

**Comment by ThomasLandauer** [comment]
@krmahadevan When I do `jar:file:///C:/1/SeleniumStandaloneServer3.2.0.jar!/javax/servlet/` I get a directory listing with about 50 files and some folders. 3 entries are starting with `LocalStrings` (each of them appearing twice):

- LocalStrings.properties
- LocalStrings_fr.properties
- LocalStrings_ja.properties

When opened, they all look similar: Everything is commented out, except something like this (from `LocalStrings.properties`):
```
err.not_iso8859_1=Not an ISO 8859-1 character: {0}
err.servlet_config_not_initialized=ServletConfig has not been initialized
value.true=true
value.false=false
```

I doubt that the .jar-file itself is corrupt, since I'm getting the same error with version 3.1.0, 3.2.0, and the newly released 3.3.0.

**Comment by krmahadevan** [comment]
@ThomasLandauer - Ok.. this is weird.. almost all of my googling on the error suggest that the `servlet-api` jar missing as the root cause. But I know that the Selenium uber jar does contain this (which you now have confirmed with your response, because those properties files come from servlet-api jar). 

FWIW, here's my java version

```
java version "1.8.0_66"
Java(TM) SE Runtime Environment (build 1.8.0_66-b17)
Java HotSpot(TM) 64-Bit Server VM (build 25.66-b17, mixed mode)
```

Maybe you could try uninstalling Java and installing it again and see if that helps ?

**Comment by ThomasLandauer** [comment]
It suddenly works, but I don't know why! ;-)

Some details:
What happened before: I have 64 Bit Windows 10. So I installed the 64 Bit Java (8 Update 121). => Selenium error message (see above)
Then I installed 32 Bit Java too. => Selenium error message (see above)
(I even started boths version explicitly from their directory in `Program Files`)

What happened today:
I tried on a different machine, so see how it looks like if it's working. => Working
Back on my main machine, I uninstalled 32 Bit Java. I was about to go ahead and uninstall 64 Bit Java too, but I thought I'll give it one last shot before re-installing => Bingo!

Thanks for your help guys!! :-)

P.S. I was convinced that there's something wrong with the Selenium download package, that's why I opened this (now offtopic) thread here.
